src/views/List.jsx

```diff
@@ -45,7 +46,15 @@ export function List({ data, listToken }) {
 
 			<ul>
 				{filteredData.map((item) => (
-					<ListItem key={item.id} name={item.name} />
+					<ListItem
+						key={item.id}
```

Comment:
ListItem gets 7 props, 4 of which are properties of item. Would passing the whole item object as a prop work in this situation? Is there a good reason not to? Genuinely curious, not asking for a change here.